csharp/Google.Protobuf.Tools.nuspec

```diff
@@ -5,7 +5,7 @@
     <title>Google Protocol Buffers tools</title>
     <summary>Tools for Protocol Buffers - Google's data interchange format.</summary>
     <description>See project site for more info.</description>
-    <version>3.7.0</version>
+    <version>3.7.0-rc1</version>
```

Comment:
https://semver.org/
C# seems also follow this: https://docs.microsoft.com/en-us/dotnet/csharp/versioning